cypress/e2e/widgets/stickers.spec.ts

```diff
@@ -86,10 +86,12 @@ function expectTimelineSticker(roomId: string) {
     // Make sure it's in the right room
     cy.get(".mx_EventTile_sticker > a").should("have.attr", "href").and("include", `/${roomId}/`);
 
-    // Make sure the image points at the sticker image
+    // Make sure the image points at the sticker image. We will briefly show it
+    // using the thumbnail URL, but as soon as that fails, we will switch to the
+    // download URL.
     cy.get<HTMLImageElement>(`img[alt="${STICKER_NAME}"]`)
         .should("have.attr", "src")
-        .and("match", /thumbnail\/somewhere\?/);
+        .and("match", /(thumbnail|download)\/somewhere/);
```

Comment:
This seems like a bad idea. It means the test won't be consistently awaiting a ready state of the app. I think the issue is the use of `.and` - https://docs.cypress.io/guides/core-concepts/retry-ability we should not be chaining like this as it doesn't retry properly.

Reply:
[5ec4026](https://github.com/matrix-org/matrix-react-sdk/pull/11440/commits/5ec402666ddef21a220cdefe2b2d09d72690fc54) searches for the alt text and URL simultaneously, so the whole thing will be retried, and it looks for the download form only, meaning we retry if the thumbnail form is visible, so the UI is in a consistent state when this succeeds.